### DISCUSSION TITLE
thread/ConcurrentHashMap

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#295] [Announcements]
# thread/ConcurrentHashMap

ConcurrentHashMap 是 Java 并发包 (java.util.concurrent) 中的一种线程安全的哈希表实现。它通过分段锁技术或更先进的并发控制技术，使得多个线程可以同时读写映射，从而提高了性能

https://javabetter.cn/thread/ConcurrentHashMap.html

**Comment by Ricehahaa** [comment]
后面的几个也太难了，真的会考吗
